pages/profile/[vin].js

```diff
@@ -1,20 +1,59 @@
+import EditCarForm from "@/components/EditCarForm";
 import Image from "next/image";
 import { useRouter } from "next/router";
+import { useState } from "react";
 import Cars from "../../db/db.json";
+
 export default function CarDetails() {
+  const [isEditing, setIsEditing] = useState(false);
   const router = useRouter();
   const { vin } = router.query;
-  const activeCar = Cars.find((car) => {
+
+  const [cars, setCars] = useState(Cars);
```

Comment:
either use all caps for constants or all all lowercase, first letter uppercase is reserved for react components and classes.

Reply:
Done!